lib/core/enums/local_settings.dart

```diff
@@ -11,7 +11,8 @@ enum LocalSettingsCategories {
   account('Account'),
   about('About'),
   debug('Debug'),
-  theming('Theming');
+  theming('Theming'),
+  videoPlayer('videoPlayer');
```

Comment:
It looks like this string is used directly in search results. It should probably be translated, but for now, can you stylize it in a more readable way, like `Video Player`?